index.html

```diff
@@ -39,14 +46,548 @@ <h1>I'm Emeka Glad to see you!</h1>
             <h5>LET'S CONNECT</h5>
 
             <ul>
-                <li><a href="#"><img src="./img/angular.jpg" alt="angular_image"></a></li>
-                <li><a href="#"><img src="./img/github.jpg" alt="github_image"></a></li>
-                <li><a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a></li>
-                <li><a href="#"><img src="./img/mee.jpg" alt="mee_image"></a></li>
-                <li><a href="#"><img src="./img//twitter.jpg" alt=""></a></li>
+
+                <li>
+
+                    <a href="#"><img src="./img/angular.jpg" alt="angular_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/github.jpg" alt="github_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img/mee.jpg" alt="mee_image"></a>
+
+                </li>
+
+                <li>
+
+                    <a href="#"><img src="./img//twitter.jpg" alt="twitter_image"></a>
+
+                </li>
+
+
+            </ul>
+
+        </div>
+
+    </section>
+
+    <section class="section2">
+
+        <figure>
+
+            <a href="#">
+
+                <img src="./img/jumbo1.jpg" alt="jumbo1_img">
+
+            </a>
+
+        </figure>
+
+        <div class="section2div">
+
+            <h3>
+
+                <a class="textdecoration font1" href="#">Tonic</a>
+
+            </h3>
```

Comment:
- [ ] The works section where you put your project cards should have a `display: grid` property and I can see that you haven't used it. Please use it.
**NOTE** You will first need to wrap all the project cards in one section and then add the `display: grid` to it. Later you will add more styles to keep mobile styles great 😃 👍